store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/IteratorSettingFactory.java

```diff
@@ -80,16 +82,18 @@ IteratorSetting getElementPostAggregationFilterIteratorSetting(final View view,
      * filtering via the Accumulo Key
      *
      * @param operation the operation
-     * @return A new {@link IteratorSetting} for an Iterator capable of filtering {@link uk.gov.gchq.gaffer.data.element.Element}s based on the options defined in the gaffer.accumulostore.operation
+     * @return A new {@link IteratorSetting} for an Iterator capable of filtering {@link uk.gov.gchq.gaffer.data.element.Element}s based on the options
+     *         defined in the gaffer.accumulostore.operation
      */
-    IteratorSetting getEdgeEntityDirectionFilterIteratorSetting(final GraphFilters operation);
+    IteratorSetting getEdgeEntityDirectionFilterIteratorSetting(final Operation operation, final View view, final DirectedType directedType);
```

Comment:
Yeah for a first attempt, if Handlers extract out this information then this should be fine.
If later when things are more clearer it makes sense to pass the whole `Operation` then this is possible.
Only as a basic idea but later the concrete implementations of this factory maybe have its own abstract `OperationBuilder` and/or its own `FieldDeclarations`

Reply:
But not at the moment.